<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>980</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GCode Executor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="topMargin">
     <number>0</number>
    </property>
    <item row="0" column="7" rowspan="4">
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="6,4">
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <property name="spacing">
          <number>4</number>
         </property>
         <item row="0" column="0" alignment="Qt::AlignTop">
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>18</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>18</height>
            </size>
           </property>
           <property name="text">
            <string>Stack</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" alignment="Qt::AlignTop">
          <widget class="QLabel" name="label_4">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>18</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>18</height>
            </size>
           </property>
           <property name="text">
            <string>Graph</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" alignment="Qt::AlignTop">
          <widget class="QLabel" name="label_5">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>18</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>18</height>
            </size>
           </property>
           <property name="text">
            <string>Dump</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QVBoxLayout" name="graphLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetMaximumSize</enum>
           </property>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QVBoxLayout" name="stackLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetMaximumSize</enum>
           </property>
          </layout>
         </item>
         <item row="1" column="2">
          <layout class="QVBoxLayout" name="dumpLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetMaximumSize</enum>
           </property>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <property name="spacing">
          <number>4</number>
         </property>
         <item row="1" column="0">
          <widget class="QTextEdit" name="txtOutput">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Output</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0" rowspan="4" colspan="7">
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="10,10">
      <property name="spacing">
       <number>4</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>456</height>
         </size>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="whatsThis">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <property name="spacing">
          <number>4</number>
         </property>
         <item row="1" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="btnRestart">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/images/Images/arrow-refresh.png</normaloff>:/images/Images/arrow-refresh.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnPrevious">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/images/Images/arrow-back.png</normaloff>:/images/Images/arrow-back.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnNext">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/images/Images/arrow-forward.png</normaloff>:/images/Images/arrow-forward.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnRun">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/images/Images/arrow-play.png</normaloff>:/images/Images/arrow-play.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnStop">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/images/Images/shape-square.png</normaloff>:/images/Images/shape-square.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTextEdit" name="txtEditorGCode">
             <property name="enabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>456</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <property name="spacing">
          <number>4</number>
         </property>
         <item row="1" column="0">
          <widget class="QTextEdit" name="txtEditorVAXCode">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="btnTranslate">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>33</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>33</height>
            </size>
           </property>
           <property name="text">
            <string>Translate to VAX Code</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/images/Images/book-go.png</normaloff>:/images/Images/book-go.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>980</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="tsmiOpen"/>
    <addaction name="separator"/>
    <addaction name="tsmiClear"/>
    <addaction name="tsmiSaveGCode"/>
    <addaction name="tsmiSaveVAXCode"/>
    <addaction name="separator"/>
    <addaction name="tsmiClose"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>&amp;Run</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Images/arrow-back.png</normaloff>Images/arrow-back.png</iconset>
    </property>
    <addaction name="tsmiRestart"/>
    <addaction name="separator"/>
    <addaction name="tsmiNext"/>
    <addaction name="tsmiPrevious"/>
    <addaction name="tsmiRun"/>
    <addaction name="tsmiStop"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="tsmiAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRun"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="tsmiOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Images/folder-open.png</normaloff>:/images/Images/folder-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="tsmiSaveGCode">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Images/disk.png</normaloff>:/images/Images/disk.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save G Code</string>
   </property>
  </action>
  <action name="tsmiSaveVAXCode">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Images/disk.png</normaloff>:/images/Images/disk.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;VAX Code</string>
   </property>
  </action>
  <action name="tsmiPrevious">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Images/arrow-back.png</normaloff>:/images/Images/arrow-back.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Previous Command</string>
   </property>
  </action>
  <action name="tsmiNext">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Images/arrow-forward.png</normaloff>:/images/Images/arrow-forward.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Next Command</string>
   </property>
  </action>
  <action name="tsmiRun">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Images/arrow-play.png</normaloff>:/images/Images/arrow-play.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Run Commands</string>
   </property>
  </action>
  <action name="tsmiStop">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Images/shape-square.png</normaloff>:/images/Images/shape-square.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Stop Execution</string>
   </property>
  </action>
  <action name="tsmiRestart">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Images/arrow-refresh.png</normaloff>:/images/Images/arrow-refresh.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Evaluate</string>
   </property>
  </action>
  <action name="tsmiAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="tsmiClose">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Images/exit.png</normaloff>:/images/Images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Close &amp;Program</string>
   </property>
  </action>
  <action name="tsmiClear">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Images/brush.png</normaloff>:/images/Images/brush.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Clear G Code</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
